.github/workflows/check-issue-links.yml

```diff
@@ -90,11 +88,12 @@ jobs:
           comment_text+="\nPlease manually unlink this PR from the following issues or reference the issues in a commit message: $extra_links"
         fi
 
-        echo $comment_text
-
-        # Export the comment text as an environment variable.
-        echo "COMMENT_TEXT=$comment_text" >> $GITHUB_ENV
+        if [[ -n "$comment_text" ]]; then
+          echo $comment_text
 
+          # Export the comment text as an environment variable.
+          echo "COMMENT_TEXT=$comment_text" >> $GITHUB_ENV
+        fi
```

Comment:
I just noticed, could you add an else case here which prints "Issues referenced in commit messages and issues linked to this PR are in sync.", just so that the logs confirm that everything is ok?